tests/integ/scala/test_dataframe_writer_suite.py

```diff
@@ -105,10 +105,6 @@ def test_write_with_target_table_autoincrement(
         Utils.drop_table(session, table_name)
 
 
-@pytest.mark.skipif(
-    "config.getoption('local_testing_mode', default=False)",
-    reason="FEAT: Inserting data into table by matching columns is not supported",
-)
 def test_negative_write_with_target_column_name_order(session):
```

Comment:
What about the test case above `test_write_with_target_column_name_order`?

Reply:
Can enable and check it out. 👍🏼